torch/onnx/_internal/exporter.py

```diff
@@ -882,29 +895,38 @@ def adapt_torch_inputs_to_onnx(
             >>> onnx_program = torch.onnx.dynamo_export(func_nested_input, x_dict, y_tuple)
             >>> print(x_dict, y_tuple)
             {'a': tensor(1.)} (tensor(2.), (tensor(3.), tensor(4.)))
-            >>> print(onnx_program.adapt_torch_inputs_to_onnx(x_dict, y_tuple, model_with_state_dict=func_nested_input))
+            >>> print(onnx_program.adapt_torch_inputs_to_onnx(x_dict, y_tuple))
             (tensor(1.), tensor(2.), tensor(3.), tensor(4.))
 
         .. warning::
             This API is experimental and is *NOT* backward-compatible.
 
         """
-        # model specified by the user has precedence, when specified
-        model_with_state_dict = model_with_state_dict or self._model_torch
-        assert (
-            model_with_state_dict is not None
-        ), "model_with_state_dict must be specified."
+        # backward compatibility: Respect the usage of *args, **kwargs for users and
+        # also None for the optional arguments
+        if (
+            "exportped_program" in model_kwargs
```

Comment:
I don't think it's possible for the below two cases. Plus there is a typo XD

Reply:
Huge mistake